packages/drizzle/src/updateMany.ts

```diff
@@ -55,12 +56,21 @@ export const updateMany: UpdateMany = async function updateMany(
 
     const table = this.tables[tableName]
 
-    const docsToUpdate = await _db
-      .select({
-        id: table.id,
-      })
-      .from(table)
-      .where(where)
+    const docsToUpdate =
```

Comment:
Could use our `chainMethods` helper, but given how small this query is, doing it like this is probably simpler

Reply:
Fine with me, chainMethods is used in other places only because the control structure didn't allow for this.